cow_amm/uni_v2_amm_lp_and_tvl_4047194

```diff
@@ -0,0 +1,94 @@
+WITH 
+cow_amm_pool as (
+select blockchain, address, token_1_address, token_2_address
+from query_3959044 q
+where address={{cow_amm_pool}}
+)
+
+, uniswap_pool as 
+    (select output_0 as pool from uniswap_v2_ethereum.Factory_call_getPair
+    where ((_0 in (select token_1_address from cow_amm_pool) and _1 in (select token_2_address from cow_amm_pool) )
+    or (_0 in (select token_2_address from cow_amm_pool) and _1 in (select token_1_address from cow_amm_pool)))
+    and not(output_0 = 0x0000000000000000000000000000000000000000)
+    limit 1
+    )
+    
+, get_lp_balance as (
+select date(evt_block_time) as day, 
+sum(case when "from" = 0x0000000000000000000000000000000000000000 THEN (value/1e18) ELSE -(value/1e18) END) as lp_supply
+from erc20_ethereum.evt_transfer
+where contract_address in (select pool from uniswap_pool)
+and ("from" = 0x0000000000000000000000000000000000000000 or "to" = 0x0000000000000000000000000000000000000000)
+group by 1
+)
+
+, total_lp as (
+select day, lp_supply, sum(lp_supply) over (order by day) as total_lp
+from get_lp_balance
+)
+
+, pair_sync_with_counts AS (
+    SELECT *
+        ,ROW_NUMBER() OVER (PARTITION BY date_trunc('day', evt_block_time) ORDER BY evt_block_time ASC ) AS sync_count_in_day
+    FROM uniswap_v2_ethereum.Pair_evt_Sync
+    WHERE contract_address in (select pool from uniswap_pool)
+    ORDER BY evt_block_time DESC
+)
+,pair_sync_last_in_day AS (
+    SELECT *, MAX(sync_count_in_day) OVER (PARTITION BY date_trunc('day', evt_block_time)) AS max_sync_count_in_day
+    FROM pair_sync_with_counts
+)
+,v2_pair_data AS (
```

Comment:
Tbh it is not clear to me what info we are trying to recover in lines 41-51 that we don't already have.